host/4/bullseye/amd64/powershell/powershell72/powershell72-slim.Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
 # Install extension bundles
 RUN apt-get update && \
     apt-get install -y gnupg wget unzip && \
-    EXTENSION_BUNDLE_VERSION_V2=2.13.0 && \
+    EXTENSION_BUNDLE_VERSION_V2=2.14.0 && \
     EXTENSION_BUNDLE_FILENAME_V2=Microsoft.Azure.Functions.ExtensionBundle.${EXTENSION_BUNDLE_VERSION_V2}_linux-x64.zip && \
     wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V2/$EXTENSION_BUNDLE_FILENAME_V2 && \
```

Comment:
Is there a reason not to update v3 extension bundle?

Reply:
There is no v3 extension bundle in the file you picked.